### DISCUSSION TITLE
I'm dumb, turned off a thing by accident, don't know what it's called (the temporary text overlay that pops up when you adjust the volume with the keyboard)

### DISCUSSION THREAD

**Discussion by dox4242** [smplayer-dev/smplayer#1096] [General]
I pressed the wrong key and the overlay doesn't display any more. I don't know which key, and I don't know what it's called to search for answers. Google and AI are dumber than I am, though (or maybe it's a GIGO situation).

**Comment by smplayer-dev** [comment]
Select menu View -> OSD -> Volume + Seek.

**Reply by dox4242** [reply]
OSD! Now I have a name. Thank you so much!
